src/server/src/main/java/io/cassandrareaper/storage/cassandra/Migration018.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2018-2018 The Last Pickle Ltd
```

Comment:
see https://github.com/thelastpickle/cassandra-reaper/pull/619/files#r259656560

Reply:
This class doesn't belong in that PR, I removed it.